infra/examples-dev/aws-msft-365/variables.tf

```
@@ -50,40 +68,52 @@ variable "psoxy_base_dir" {
     condition     = can(regex(".*\\/$", var.psoxy_base_dir))
     error_message = "The psoxy_base_dir value should end with a slash."
   }
-  validation {
-    condition     = can(regex("^[^~].*$", var.psoxy_base_dir))
-    error_message = "The psoxy_base_dir value should be absolute path (not start with ~)."
-  }
 }
 
-variable "caller_gcp_service_account_ids" {
+variable "pseudonymize_app_ids" {
+  type        = string
+  description = "if set, will set value of PSEUDONYMIZE_APP_IDS environment variable to this value for all sources"
+  default     = false
+}
+
+variable "enabled_connectors" {
   type        = list(string)
-  description = "ids of GCP service accounts allowed to send requests to the proxy (eg, unique ID of the SA of your Worklytics instance)"
-  default     = []
+  description = "list of ids of connectors to enabled; see modules/worklytics-connector-specs"
 
-  validation {
-    condition = alltrue([
-      for i in var.caller_gcp_service_account_ids : (length(regexall("^\\d{21}$", i)) > 0)
-    ])
-    error_message = "The values of caller_gcp_service_account_ids should be 21-digit numeric strings."
-  }
+  default = [
+    "azure-ad",
+    "outlook-cal",
+    "outlook-mail",
+    "asana",
+    "hris",
+    "slack-discovery-api",
+    "zoom",
```

Just if you could keep dropbox as it was before:

```suggestion
    "azure-ad",
    "dropbox-business",
    "outlook-cal",
    "outlook-mail",
    "asana",
    "hris",
    "slack-discovery-api",
    "zoom",
```